### PR TITLE
Clients section

### PR DESCRIPTION
**Pull request by GabrielBFurlani**
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -188,6 +188,217 @@ <h2 class="featurette-heading-home">
...
+
+            <br />
+            React portable application <br />
+            Native who went public <br />
+            at the end of 2019, available <br />
+            for Android and iOS, is <br />
+            responsible for managing <br />
+            plans, buying packages, <br />
+            account view, <br />
+            among others. <br />
```
Comment (victor-accarini) [inline index.html:307]: React Native mobile app who went public at the end of 2019, available for Android and iOS, is responsible for managing the user plans, account information, buying packages, among others.

File: index.html [hunk 2]
```diff
@@ -188,6 +188,217 @@ <h2 class="featurette-heading-home">
...
+            Web applications run  <br/>
+            on the interactive TVs of  <br/>
+            Shopping Iguatemi stores  <br/>
+            (São Paulo) and Londres.  <br/>
+            One shows the history of  <br/>
+            brand and information of a  <br/>
+            product via QR Code, the other  <br/>
+            allows choice of colors and  <br/>
+            showed a combination of  <br/>
+            products
```
Comment (victor-accarini) [inline index.html:380]: Web applications executed on the interactive TVs of Shopping Iguatemi (São Paulo) and Londres stores. One shows the history of the brand and the other allows choice of colors, showed a combination of products and information of a product via QR Code.

File: index.html [hunk 3]
```diff
@@ -457,7 +668,7 @@ <h2 class="featurette-heading-social-media">
 
     <!-- FOOTER -->
     <footer class="low-footer">
-       &copy; 2016 - 2021 | ProFUSION Sistemas e Soluções LTDA
+      &copy; 2016 - 2021 | ProFUSION Sistemas e Soluções LTDA
```
Comment (victor-accarini) [inline index.html:671]: O linter não esta pegando essas idêntações? Ou isso foi feito antes de adicionar o linter?

**Review by victor-accarini — COMMENTED** [review]
Remove all company logos since we need their permission to use it, in their place use a gray or white div with the correct dimensions.

File: index.html (hunk 3 above)
Comment (GabrielBFurlani) [reply index.html:680]: Estranho, acho que foi depois, vou conferir aqui

File: index.html (hunk 2 above)
Comment (victor-accarini) [reply index.html]: @GabrielBFurlani be careful, the text that I sent is not the same that you updated. Always double check

Comment (GabrielBFurlani) [reply index.html]: It really lacked attention! thanks

**Review by victor-accarini — APPROVED** [review]

**Pull request merged**